Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -96,6 +96,21 @@ public void SerializesAndDeserializesCustomPropertyNames()
             Check(TestObjects.fmRadio);
         }
 
+        [TestMethod]
+        public void SerializesAndDeserializesGetterAndSetterMethods()
+        {
+            var serializer = new IonSerializer();
+
+            var stream = serializer.Serialize(TestObjects.honda);
+            IIonStruct serialized = StreamToIonValue(stream);
+            Assert.IsTrue(serialized.ContainsField("color"));
+
+            stream.Position = 0;
+            var deserialized = serializer.Deserialize<Car>(stream);
+            Assert.IsNotNull(deserialized.color);
```

Comment:
Do you need this IsNotNull check given the subsequent AreEqual check?

Reply:
Removed.